mungegithub/cherrypick/deployment.yaml

```diff
@@ -86,11 +27,16 @@ spec:
           limits:
             cpu: 600m
         volumeMounts:
+        - mountPath: /etc/config
```

Comment:
I used "/etc/config" so that the config path is consistent across all mungegithub apps (submit-queue, cherrypick, publisher).  Is there an advantage to having each app deploy to a different path or do you just want a more descriptive name than "config".  If the latter is the case then "/etc/munge-config" might be best since that could be used for all apps.